data/json/recipes/recipe_others.json

```diff
@@ -56,7 +56,7 @@
   {
     "type": "recipe",
     "activity_level": "LIGHT_EXERCISE",
-    "result": "bodypillow",
+    "result": "bodypillow_makeshift",
```

Comment:
You need to obsolete the original recipe as well, this breaks it for people who update.
```
{
    "type": "recipe",
    "result": "bodypillow",
    "obsolete": true
  }
  ```
  Add to recipes/recipe_obsolete.json

Reply:
are sure about that? I thought it only broke if items were changed; oh wait probably if someone was in the process of crafting one while updating; thanks for already giving me what I need for that file <3